ts/torch_handler/base_handler.py

```diff
@@ -53,6 +53,10 @@
     )
     PT2_AVAILABLE = False
 
+if packaging.version.parse(torch.__version__) > packaging.version.parse("2.1.1"):
+    PT220_AVAILABLE = True
```

Comment:
does this mean pt.2.2? Can we call this call PT2_2-0 should be easier to read

Reply:
Doesn't allow `PT2_2-0`